### ISSUE TITLE
Missing and old links to Accessibility Properties for Schema.org

### ISSUE THREAD

**Issue by clapierre** [w3c/a11y-discov-vocab#182]
The old URL "[https://www.w3.org/2021/a11y-discov-vocab/latest/](https://www.w3.org/2021/a11y-discov-vocab/latest/" redirects to "[https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/](https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/))" 
We need to update all our URLs to point to the Dec 09, 2024 Final Community Group Report.

Also missing is links for accessibilityFeature and accessibilityHazard

**Comment by mattgarrish** [comment]
I'm not sure I understand this issue. Why do you call the latest link the "old url"?

The latest link gets updated (by us, not automatically) whenever we publish to redirect users to the new version. If you use the dated published links, you're going to have to update those links every time we publish, as well.

**Comment by mattgarrish** [comment]
For reference, the latest URL points to this file: https://github.com/w3c/a11y-discov-vocab/blob/main/latest/index.html

**Comment by clapierre** [comment]
ok thanks @mattgarrish I will keep the old link which redirects to the latest.  I just saw a 2021 date in the URL and then saw it redirecting and thought I should change to where it was redirecting to.  So, I will only add the missing links for accessibilityFeature and Hazard then.

**Comment by mattgarrish** [comment]
Ya, that's just the way W3C works at times. Because we created the latest link redirect in 2021 the URL has that year in it. It's not actually pointing to a published document but redirects to the one I linked to above and then that one gets you to the right file. That was the best we could do to avoid becoming dependent on W3C staff to keep correcting the URL every time we publish.

If you don't care about using a W3C URL, you could bypass the first redirect and use:
https://w3c.github.io/a11y-discov-vocab/latest/

We use the formal W3C ones for the links in the document headers so they look more official.
